pkg/payload/payload.go

```diff
@@ -134,13 +133,19 @@ type metadata struct {
 }
 
 func LoadUpdate(dir, releaseImage, excludeIdentifier string, includeTechPreview bool, profile string,
-	capabilities capability.ClusterCapabilities) (*Update, error) {
+	knownCapabilities []configv1.ClusterVersionCapability) (*Update, error) {
 
 	payload, tasks, err := loadUpdatePayloadMetadata(dir, releaseImage, profile)
 	if err != nil {
 		return nil, err
 	}
 
+	// We only pass known capabilities at payload load time to avoid doing any capability
+	// enable filtering which only occurs during apply.
+	onlyKnownCaps := configv1.ClusterVersionCapabilitiesStatus{
+		EnabledCapabilities: knownCapabilities,
+		KnownCapabilities:   knownCapabilities}
```

Comment:
`EnabledCapabilities: knownCapabilities` does not look right. At this point we do not know what is enabled, do we?